ui/src/shared/components/AutoRefresh.js

```diff
@@ -27,6 +27,11 @@ const AutoRefresh = ComposedComponent => {
     }
 
     componentWillReceiveProps(nextProps) {
+      const notInViewDidUpdate = !_.isEqual(
```

Comment:
* If these are booleans, can we use a `!==` instead?
* Can we phrase this variable name in the positive, throughout? `inView` is easier to reason about than `notInView`, simply by avoiding double negatives